src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -96,20 +160,14 @@ public boolean hasPerson(Integer personId) {
         return participantIds.contains(personId);
     }
 
-    /**
-     * Invite a person into the activity.
-     * @param personId ID of the person to be invited.
-     */
-    public void invite(Integer personId) {
-        participantIds.add(personId);
-    }
-
     /**
      * Remove people from the activity
-     * @param personId The people that will be removed from the activity.
+     * @param people The people that will be removed from the activity.
      */
-    public void disinvite(Integer personId) {
-        participantIds.remove(personId);
+    public void disinvite(Person ... people) {
+        // haven't implemented what if list does not contain that specific person
+        // TODO: also care about transfermatrix and debtmatrix
```

Comment:
Let's leave this for @JohnNzj to complete when he sets up the disinvite command.